packages/flutter_tools/lib/src/resident_web_runner.dart

```diff
@@ -58,12 +62,16 @@ class ResidentWebRunner extends ResidentRunner {
   }
 
   @override
-  Future<void> cleanupAfterSignal() {
+  Future<void> cleanupAfterSignal() async {
+    await _connection.sendCommand('Browser.close');
```

Comment:
nit: do we usually capitalize command names?

Reply:
This is a webkit inspection protocol command